src/commands/miscellaneous/RemoveGoStudy.ts

```diff
@@ -0,0 +1,103 @@
+import { GuildPreferencesCache } from "@/redis";
+import type { DiscordClient } from "@/registry/DiscordClient";
+import BaseCommand, {
+	type DiscordChatInputCommandInteraction
+} from "@/registry/Structure/BaseCommand";
+import Logger from "@/utils/Logger";
+import {
+	PermissionFlagsBits,
+	SlashCommandBuilder
+} from "discord.js";
+
+export default class RemoveGoStudyCommand extends BaseCommand {
+	constructor() {
+		super(
+			new SlashCommandBuilder()
+				.setName("remove_gostudy")
+				.setDescription(
+					"Remove the Forced Mute role. (for mods)"
+				)
+				.addUserOption((option) =>
+					option
+						.setName("user")
+						.setDescription("User to remove forced mute from")
+						.setRequired(true)
+				)
+				.setDMPermission(false)
+		);
+	}
+
+	async execute(
+		client: DiscordClient<true>,
+		interaction: DiscordChatInputCommandInteraction<"cached">
+	) {
+		let user = interaction.options.getUser("user", false) || interaction.user;
```

Comment:
The user is a required field, and even if it weren't it doesn't make sense for the mod who ran he command to remove gostudy from themselves